docs/user/index.rst

```diff
@@ -157,7 +158,7 @@ We have a few places for you to get started:
 .. descriptions here are active
 
 :doc:`/tutorial/index`
-  Take the first practical steps with Read the Docs.
+  Take the first steps with Read the Docs in 30 minutes.
```

Comment:
This implies to me that you can't do anything in less than 30 minutes, which isn't ideal.

```suggestion
  Follow the Read the Docs tutorial.
```